src/url.zig

```diff
@@ -1,5 +1,8 @@
-// This is close to WHATWG URL, but we don't want the validation errors
+/// Unsafe parser. Assumes the given string is already a valid URL.
+///
+/// Use bun.jsc.URL for a safe, compliant URL parser.
 pub const URL = struct {
+    // This is close to WHATWG URL, but we don't want the validation errors
```

Comment:
```suggestion
/// Use bun.jsc.URL for a safe, compliant URL parser.
// This is close to WHATWG URL, but we don't want the validation errors
pub const URL = struct {
```

This makes it seem like the comment applies to `log` specifically